src/constants.ts

```diff
@@ -3,7 +3,7 @@ import { ObsidianGitSettings } from "./types";
 
 export const DEFAULT_SETTINGS: ObsidianGitSettings = {
     commitMessage: "vault backup: {{date}}",
-    autoCommitMessage: undefined, // default undefined for settings migration
+    autoCommitMessage: "vault backup: {{date}}",
```

Comment:
No I would rather remove the field here and use `Omit<ObsidianGitSettings,"autoCommitMessage">` as type for `DEFAULT_SETTINGS`. `autoCommitMessage` should be set to the current `commitMessage` value, in case it's undefined. That's what's done in `migrateSettings`.

Reply:
@Vinzent03 
OK, I fixed again.
cc146da